src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -34,13 +43,31 @@ private static void readConfigFile(String configFile) throws ApplicationExceptio
             throw new ApplicationException("Configuration file is invalid", ex);
         }
         JSONObject obj = new JSONObject(jsonString);
-        dbPort = obj.getJSONObject("parameters").getJSONObject("db").getString("port");
-        dbHost = obj.getJSONObject("parameters").getJSONObject("db").getString("host");
-        dbUser = obj.getJSONObject("parameters").getJSONObject("db").getString("user");
-        dbPassword = obj.getJSONObject("parameters").getJSONObject("db").getString("#password");
-        dbName = obj.getJSONObject("parameters").getJSONObject("db").getString("database");
-        query = obj.getJSONObject("parameters").getString("query");
-        outputFile = obj.getJSONObject("parameters").getString("outputFile");
+        JSONObject params = obj.getJSONObject("parameters");
+        JSONObject db = params.getJSONObject("db");
+        dbPort = db.getString("port");
+        dbHost = db.getString("host");
+        dbUser = db.getString("user");
+        dbPassword = db.getString("#password");

```

Comment:
also i think we need some docs for the json file spec, it's getting non-trivial

Reply:
j, I'll put it in the readme